staging/src/k8s.io/legacy-cloud-providers/aws/aws_loadbalancer.go

```diff
@@ -145,9 +145,17 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 			}
 		}
 
+		var privateIPs []string
+		if privateIPList, present := annotations[ServiceAnnotationLoadBalancerPrivateIPs]; present {
```

Comment:
It is better if we add validation to check this annotation is allowed only for "internal" load balancers.

Reply:
Since we are modifying this code block, lets also validate that EIP annotation can only be specified for "internet-facing" or public load balancers.